int32 control_mode
float32 first
float32 second
float32 third